README.md

```diff
@@ -42,11 +42,12 @@ Requirements:
 
 * PostgreSQL 9.5 or newer
 * Ruby 2.5.0
+* [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/)
 
 ### Setup on Ubuntu
 ```bash
 # Install required packages
-$ sudo apt-get install postgresql libpq-dev libcurl3 libcurl3-gnutls libcurl4-openssl-dev postgresql-contrib-9.5
+$ sudo apt-get install postgresql libpq-dev libcurl3 libcurl3-gnutls libcurl4-openssl-dev postgresql-contrib-9.5 chromium-chromedriver
```

Comment:
Were you able to test this on a Linux machine? I cannot verify this until Sunday since I only got a MBP with me right now.

Reply:
No, did not try this out on a Linux machine, just on a trusty Ubuntu Docker container some weeks ago.